### PR TITLE
Feature: Fingertips slides

### PR DESCRIPTION
**Pull request by JakubFrackowiak**
Resolves: #129 
image
image

Here is slider in action:
https://youtu.be/eLuLoT6jyoc

### CODE REVIEW

File: components/BusinessHeroes/BuisnessHeroesSlides.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  StorageImage,
+  useFirestore,
+  useFirestoreCollectionData,
+} from "reactfire"
+import { collection, limit, query } from "firebase/firestore"
+import { Hero } from "types/hero"
+import { Stack } from "@mui/material"
+import { useEffect, useRef, useState } from "react"
+
+const delay = 5000
```
Comment (OliwierOST) [inline components/BusinessHeroes/BuisnessHeroesSlides.tsx:11]: I don't think there is a need to make the delay a variable. Why don't just insert the value into the timeout directly?

File: components/BusinessHeroes/BuisnessHeroesSlides.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+export function BuisnessHeroesSlides() {
```
Comment (OliwierOST) [inline components/BusinessHeroes/BuisnessHeroesSlides.tsx:13]: Buisness? What does that mean?

**Review by OliwierOST — DISMISSED** [review]

**Comment by piotrostr** [comment]
also please include some padding, otherwise it looks a bit crammed

File: components/BusinessHeroes/BuisnessHeroesSlides.tsx (hunk 2 above)
Comment (JakubFrackowiak) [reply components/BusinessHeroes/BuisnessHeroesSlides.tsx:13]: I am sure my keyboard lagged lol

File: components/BusinessHeroes/BuisnessHeroesSlides.tsx (hunk 1 above)
Comment (JakubFrackowiak) [reply components/BusinessHeroes/BuisnessHeroesSlides.tsx:11]: The delay variable is necessary for it to work, otherwise it doesn't.

**Comment by JakubFrackowiak** [comment]
I encountered some issues probably with fetching data while making requested changes, so dont merge it untill I fix those please. Shouldn't be too long.

**Comment by JakubFrackowiak** [comment]
All good i suppose

File: components/BusinessHeroes/BusinessHeroesSlides.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import {
+  StorageImage,
```
Comment (piotrostr) [inline components/BusinessHeroes/BusinessHeroesSlides.tsx:2]: if importing a type do `import { type StorageImage } from "reactfire";`

**Review by piotrostr — APPROVED** [review]

File: components/BusinessHeroes/BusinessHeroesSlides.tsx (hunk 3 above)
Comment (JakubFrackowiak) [reply components/BusinessHeroes/BusinessHeroesSlides.tsx:2]: storage image is not a type, also I tried to import it as a type and vscode underlines it obvliously.

Comment (JakubFrackowiak) [reply components/BusinessHeroes/BusinessHeroesSlides.tsx:2]: image

Comment (piotrostr) [reply components/BusinessHeroes/BusinessHeroesSlides.tsx:2]: oh true that, sorry my bad

**Review by piotrostr — APPROVED** [review]

**Pull request merged**